cabal-install/src/Distribution/Client/Utils.hs

```diff
@@ -362,8 +360,4 @@ progressMessage verbosity phase subject = do
         ProgressCompleted   -> "Completed    "
 
 cabalInstallVersion :: Version
-#ifdef __DOCTEST__
-cabalInstallVersion = mkVersion [3,3]
-#else
-cabalInstallVersion = mkVersion' Paths_cabal_install.version
-#endif
+cabalInstallVersion = mkVersion [3,5]
```

Comment:
Hmm we probably need to add this to the release checklist (can be in another pr)

Reply:
Added